libraries/lambda_handlers/ts_v2_catalog_handler.py

```diff
@@ -448,8 +448,8 @@ def _index_words_files(self, lid, rid, format, process_id):
         :param format:
         :return:
         """
-        word_title_re = re.compile('^#([^#]*)#?', re.UNICODE)
-        h2_re = re.compile('^##([^#]*)#*', re.UNICODE)
+        word_title_re = re.compile('^#([^#\n]*)#*', re.UNICODE)
+        h2_re = re.compile('^##([^#\n]*)#*', re.UNICODE)
```

Comment:
Why do you exclude newline characters from the match?  i don't think \# headers in Markdown are allowed to span multiple line.

Reply:
Newlines were being captured, and this excludes them.
Here's a demo of how it was going wrong before: https://repl.it/@Albeit/TwMdParseError
Just to clarify though, this fix isn't my code; it's being cherry-picked from uW.

Reply:
Oh, I understand now, I didn't realize the capture was multiline, and this forces it to respect newlines for H2.  Thanks for explaining and for the concrete example!